components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/internal/services/TemplateManagerService.java

```diff
@@ -107,7 +105,7 @@ public void deleteBusinessRule(String uuid) { // todo: verify next lower level
             foundBusinessRule = findBusinessRuleFromTemplate(uuid);
         } catch (TemplateManagerException e) {
             // No Business Rule Found
-            log.error(e); // todo: (Q) is this ok?
+            log.error(e.toString()); // todo: (Q) is this ok?
```

Comment:
Please provide more context info for error. Check other places as well.

Reply:
Reimplemented by b181384b3c4bdd1b2ac145955cb54a71628805e7